### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by seralra96**

### CODE REVIEW

**Comment by juusechec** [comment]
Can I merge this changes replacing translations?

**Comment by seralra96** [comment]
@juusechec  It's ready for merge, i resolved the conflicts keeping the translations files from the PR branch. This avoids loose the translations

File: src/toolbar.rst [hunk 1]
```diff
@@ -29,6 +49,11 @@ Fill More BFS and Less
 You can fill More BFS and Less topology table using the button *Fill More BFS
 and Less* on Toolbar LADM_COL which makes automatic and faster this job.
 
+This are the tables that relate with this function:
+
+masccl --> puntolindero and lindero
+menos --> puntolindero and lindero (holes or intern rings)
```
Comment (gacarrillor) [inline src/toolbar.rst]: Terreno and Lindero

Comment (seralra96) [reply src/toolbar.rst]: fixed https://github.com/AgenciaImplementacion/Asistente-LADM_COL-docs/pull/9/commits/d670a21d96504dd2dbdc3ce41c16f7b3bcb76255

**Pull request merged**